src/main/java/io/candydoc/ddd/annotations/Factory.java

```diff
@@ -0,0 +1,16 @@
+package io.candydoc.ddd.annotations;
+
+import java.lang.annotation.Documented;
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Inherited;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+@Documented
+@Inherited
+@Target({ElementType.TYPE})
+@Retention(RetentionPolicy.RUNTIME)
+public @interface Factory {
```

Comment:
In order to distinguish it from a Factory of non-domain models, do you think it's relevant to rename it to `DomainFactory` ?

Reply:
We can rename it to `DomainFactory` as we did with `DomainEntity` and `DomainRepository`. But, to be consistent, shouldn't we add `Domain` to all candy-doc annotations ?

Reply:
I think it would not make any sense for ValueObject and Aggregate to add a `Domain` prefix.